src/spec.js

```diff
@@ -28,3 +28,86 @@ test( 'doubleEach', function ( test ) {
   test.end();
 });
 
+
+test ( 'Animal', function ( test ) {
+  const animal = new Animal();
+
+	test.ok( animal instanceof Animal, 'should create an instance of Animal' );
+
+	test.end();
+});
+
+test ( 'Animal.speak', function ( test ) {
+  const animal = new Animal();
+
+  const actual = animal.speak;
+  const expected = 'generic sound';
+
+  test.equal( actual, expected, 'the animal should speak');
+
+  test.end();
+});
+
+test ( 'animal inheritance', function ( test ) {
+
+  let actual, expected;
+
+  const reptile = new Reptile();
+
+  test.ok( reptile instanceof Reptile, 'should be an instance of Reptile' );
+  test.ok( reptile instanceof Animal, 'should be an instance of Animal' );
+
+  expected = Reptile.SOUND; //not sure what Reptile.SOUND is
```

Comment:
This isn't testing what you think it's testing. :-)

In this case, both the expected and actual values are `undefined`, so they are "equal" and the test passes. But the app is not performing the way it should. In your `speak` methods, you are logging out the text; what if you returned the text instead?